care/emr/resources/patient_identifier/spec.py

```
@@ -0,0 +1,77 @@
+    use: PatientIdentifierUse
+    description: str = ""
+    system: str
+    required: bool
+    unique: bool
+    regex: str
+    display: str
+    retrieve_config: PatientIdentifierRetrieveConfig = {}
+    default_value: str | None = None
+
+    @field_validator("default_value")
+    def validate_default_value(cls, v):
```

Ruff (0.11.9)

First argument of a method should be named `self`

Rename `cls` to `self`

(N805)